pages/linux/ldapsearch.md

```diff
@@ -0,0 +1,35 @@
+# ldapsearch
+
+> CLI utility for querying an LDAP directory.
+
+- Query LDAP for all items that match filter "{{memberOf}}={{group1}}" and return {{displayName}}:
+
+`ldapsearch -D '{{admin_DN}}' -w '{{password}}' -h {{ldap_host}} -b {{base_ou}} '{{memberOf}}={{group1}}' {{displayName}}`
+
+- Query LDAP with a no-newline password file and return {{displayName}}:
+
+`ldapsearch -D '{{admin_DN}}' -y '{{password_file}}' -h {{ldap_host}} -b {{base_ou}} '{{memberOf}}={{group1}}' {{displayName}}`
+
+- Query LDAP for up-to 5 items that match filter "{{memberOf}}={{group1}}" and return {{displayName}}:
```

Comment:
Perhaps this will read better?
```suggestion
- Return the first 5 items that match the given filter:
```

Reply:
@sbrl my read of LDAP docs suggests that results aren't sorted by default, I committed your change but I'm worried that this could be misleading.

Reply:
I see. What about this then?
```suggestion
- Return 5 items that match the given filter:
```

Reply:
LGTM, thank you!